### PR TITLE
Feature/event logics

### PR DESCRIPTION
**Pull request by LFSCamargo**

### CODE REVIEW

File: server/src/modules/event/eventModel.ts [hunk 1]
```diff
@@ -0,0 +1,94 @@
+import * as mongoose from 'mongoose';
+
+export interface Event extends mongoose.Document {
```
Comment (rodrigooler) [inline server/src/modules/event/eventModel.ts]: @LFSCamargo run prettier

File: server/src/modules/event/eventModel.ts [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+  limit: number,
+  location: {
+    type: string,
+    coordinates: number[],
+  },
+  date: string,
+}
+
+const Schema = new mongoose.Schema({
+  owner: {
```
Comment (rodrigooler) [inline server/src/modules/event/eventModel.ts]: always add the default to have the schema 100% equal to all other documents

File: server/src/modules/event/eventModel.ts [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+  }
+}, {
+  timestamps: {
+    createdAt: 'createdAt',
+    updatedAt: 'updatedAt',
+  },
+  collection: 'event',
+});
+
+export default mongoose.model<Event>('Event', Schema);
```
Comment (rodrigooler) [inline server/src/modules/event/eventModel.ts]: add new blank line

File: server/src/modules/event/eventResolvers.ts [hunk 4]
```diff
@@ -0,0 +1,265 @@
+import userModel, { Roles } from '../user/userModel'
+import eventModel, { Event } from './eventModel';
+import { GraphQLContext } from '../../graphqlTypes';
+
+interface AddInput {
```
Comment (rodrigooler) [inline server/src/modules/event/eventResolvers.ts]: would not it be better to have a type-only file?

example:

eventResolverTypes.ts

File: server/src/modules/event/eventModel.ts (hunk 1 above)
Comment (LFSCamargo) [reply server/src/modules/event/eventModel.ts]: its not running on pre-commit?

**Pull request merged**